app/views/schools/index.html.erb

```diff
@@ -5,6 +5,9 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl"><%= title %></h1>
 
+    <p class="govuk-body">You have access to manage ordering on behalf of more than one organisation.</p>
+    <p class="govuk-body">Select which one you want to view:</p>
```

Comment:
I know you were a little worried about not including this in the locales file but I don't think you need to stress about it because it's very unlikely we will reuse this copy on other pages and if you look at the other pages you will see a lot of content isn't in the locale files. We tend to use a locales file if we likely to reuse the copy. For example, keeping browser title,  page headings, and breadcrumbs consistent then locales are great to use.